plugins/hydra_optuna_sweeper/news/1906.feature

```diff
@@ -0,0 +1 @@
+Add 'custom_search_space' configuration option to allow extending trial objects programmatically
```

Comment:
```suggestion
Add experimental 'custom_search_space' configuration node to allow extending trial objects programmatically.
```

I think the word "option" could be confusing because it has a different meaning in the context of Hydra's defaults list (see e.g. ["Terminology - Config Group Option"](https://hydra.cc/docs/advanced/terminology/#config-group-option)).